src/Database/Connection.php

```diff
@@ -185,7 +184,7 @@ public function connect()
             $this->_driver->connect();
 
             return true;
-        } catch (Exception $e) {
+        } catch (\Exception $e) {
```

Comment:
Why is this necessary again? It should be OK to not use FQCN inside code and always include them via use statements.